### ISSUE TITLE
backend generator?

### ISSUE THREAD

**Issue by gortok** [statiqdev/CleanBlog#1]
Do you have a sample backend generator for this?

**Comment by daveaglick** [comment]
So close to shipping! The idea is that this is a [theme](https://statiq.dev/web/themes/) so the files here should be copied (or cloned) into a "theme" folder alongside your "input" folder. Statiq Web knows about the theme folder and anything in "theme/input" will underlap your "input" folder (I.e. the virtual file system that Statiq sees will consist of everything in both folders with files in "input" overriding those in "theme/input"). If you want to customize the theme, you can do so by editing the theme files in "theme/input" or by copying them to "input" and editing it there.

In the process of building out this theme I've found a lot of little gaps and have been updating Statiq Framework and Statiq Web as I go. First time's a charm I guess. That does mean this first theme isn't actually usable until the new versions of Statiq Framework and Statiq Web ship, which will happen literally any day now. I just have a couple finishing touches. Once this theme is out, then I'm planning on releasing several more in quick succession since the mechanics will have been settled.

I'm also planning on updating my own blog shortly after this theme ships, so that can serve as a good complete example.

TL;DR:
- Very soon (possibly even tonight) new versions of Statiq Framework and Statiq Web will be released
- At that point, this theme will also be officially released
- You'll be able to create a Statiq Web app, copy/clone the theme into a "theme" folder, and be up and running with a blog or site just by adding Markdown files under "input/posts".
- More themes will follow, along with additional enhancements to this theme (series of posts is something I want to get working relatively quickly for example)

**Comment by gortok** [comment]
This all sounds great.  I wanted to have statiq host the LWIDN newsletter (https://www.lastweekin.net) and that's why I'm asking all these questions.   I'll keep an eye out.
